README.md

```diff
@@ -2,43 +2,75 @@
 
 [![License](https://img.shields.io/github/license/KIT-MRT/arbitration_graphs)](./LICENSE)
 
-Arbitration graphs combine simple atomic behavior components into more complex behaviors for decision-making and behavior generation
+**Hierarchical behavior models for complex decision-making and behavior generation in robotics!**
```

Comment:
I like it! We should change the project description in the sidebar accordingly.

Reply:
Good point!